src/pyshark/capture/file_capture.py

```diff
@@ -26,14 +27,15 @@ def __init__(self, input_file=None, keep_packets=True, display_filter=None, only
         :param decryption_key: Optional key used to encrypt and decrypt captured traffic.
         :param encryption_type: Standard of encryption used in captured traffic (must be either 'WEP', 'WPA-PWD', or
         'WPA-PWK'. Defaults to WPA-PWK).
-        :param tshark_path: Path of the tshark binary
         :param decode_as: A dictionary of {decode_criterion_string: decode_as_protocol} that are used to tell tshark
         to decode protocols in situations it wouldn't usually, for instance {'tcp.port==8888': 'http'} would make
         it attempt to decode any port 8888 traffic as HTTP. See tshark documentation for details.
+        :param tshark_path: Path of the tshark binary
+        :param override_prefs: A dictionary of tshark preferences to override, {PREFERENCE_NAME, PREFERENCE_VALUE, ...}.
```

Comment:
Docs should be {PREFERENCE_NAME: PREFERENCE_VALUE, ..} rather than a comma between them